lib/src/model/knowledge_panels.dart

```diff
@@ -38,4 +38,6 @@ class KnowledgePanels {
     }
     return result;
   }
+
+  static const String simplifiedRootId = 'knowledge_panels';
```

Comment:
```suggestion
  static const String simplifiedRootId = 'simplified_root';
```
Or something like that I guess?

Reply:
Yes sorry I made a mistake, thank you for catching this